tests/unit/samplegen/common_types.py

```diff
@@ -41,7 +41,12 @@
 DummyMessage.__new__.__defaults__ = (False,) * len(DummyMessage._fields)
 
 DummyField = namedtuple("DummyField",
-                        ["message", "enum", "repeated", "field_pb", "meta"])
+                        ["message",
+                         "enum",
+                         "repeated",
+                         "field_pb",
+                         "meta",
+                         "is_primitive"])
```

Comment:
nit: All the other fields are nouns, but this one is a verb. I'd suggest changing it to `primitive` or `primitive_type`.

Reply:
This is one where my hands are mostly tied: it's a duck-type mask over the `wrappers.Field.is_primitive` property. The dummy type exists because in most cases constructing a real `Field` makes the test case bloated.

I could change the name of the real property, but that effects a lot of other code too.